### ISSUE TITLE
how-to:Use secrets building block retrieves the secret from the Map structure

### ISSUE THREAD

**Issue by dybxin** [dapr/components-contrib#533]
> /area runtime
> /area docs

## Ask your question here
how-to:Use secrets building block retrieves the secret from the Map structure?

local-secret-store.yaml
```yaml
apiVersion: dapr.io/v1alpha1
kind: Component
metadata:
  name: localsecretstore
  namespace: default
spec:
  type: secretstores.local.file
  metadata:
  - name: secretsFile
    value: secrets.json
  - name: nestedSeparator
    value: ":"
```

secrets.json
```json
{
    "mysecret": "abcd",
    "connectionStrings": {
        "sql": "your sql connection string",
        "mysql": "your mysql connection string"
    }
}
```
after `dapr run `, i use the flattened key to access the secret. 

```
localhost:3500/v1.0/secrets/localsecretstore/connectionStrings:sql
```
i got errorCode: `ERR_STATE_GET`

**Comment by dybxin** [comment]
@youngbupark i'm sorry, Did anyone follow up on this issue?

**Comment by cmendible** [comment]
Hi @dybxin this was fixed in [Dapr Runtime v1.0.0-rc.2](https://github.com/dapr/dapr/releases/tag/v1.0.0-rc.2) issue https://github.com/dapr/components-contrib/issues/550

If you have not upgraded as a workaround you can remove the `nestedSeparator` metadata from the secret store definition and it will use the default `:`.

@youngbupark can you close this one?

**Comment by youngbupark** [comment]
closed by https://github.com/dapr/components-contrib/pull/551
